lib/Elastica/Filter/GeoBoundingBox.php

```diff
@@ -20,32 +20,34 @@ class GeoBoundingBox extends AbstractFilter
      *
      * @param string $key         Key
      * @param array  $coordinates Array with top left coordinate as first and bottom right coordinate as second element
+     * @param array  $positions array sets the bounding box points
      */
-    public function __construct($key, array $coordinates)
+    public function __construct($key, array $coordinates, array $positions = ['top_left', 'bottom_right'])
     {
-        $this->addCoordinates($key, $coordinates);
+        $this->addCoordinates($key, $coordinates, $positions);
     }
 
     /**
      * Add coordinates.
      *
      * @param string $key         Key
      * @param array  $coordinates Array with top left coordinate as first and bottom right coordinate as second element
+     * @param array  $positions array sets the bounding box points
      *
      * @throws \Elastica\Exception\InvalidException If $coordinates doesn't have two elements
      *
      * @return $this
      */
-    public function addCoordinates($key, array $coordinates)
+    public function addCoordinates($key, array $coordinates, array $positions)
```

Comment:
We should also set a default here, same as in constructor. Otherwise it is not BC as the third param is required.